rocketpy/AeroSurface.py

```diff
@@ -202,15 +211,27 @@ def kind(self):
 
     @kind.setter
     def kind(self, value):
-        # Analyze type
+        # Analyzes nosecone type
+        # Sets the k for Cp calculation
+        # Sets the function which creates the respective curve
         self._kind = value
+        value = (value.replace(" ", "")).lower()
+
         if value == "conical":
             self.k = 2 / 3
-        elif value == "ogive":
-            self.k = 0.466
+            self.y_nosecone = Function(lambda x: x * self.baseRadius / self.length)
```

Comment:
The `y_nosecone` variable name is not clear enough, since it is becoming an attribute of the class.

I would like if we could think in a better name together.